www/app/AppKernel.php

```diff
@@ -22,6 +22,7 @@ public function registerBundles()
             new Braincrafted\Bundle\BootstrapBundle\BraincraftedBootstrapBundle(),
             new Ecwec\Bundle\CoreBundle\CoreBundle(),
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
+            new Ecwec\Bundle\WeatherDataProviderBundle\WeatherDataProviderBundle(),
```

Comment:
I just really don't like the WeatherDataProviderBundle name. Generally speaking, Providers have a different meaning in the Symfony world. 

However, I do not have a better name than the one you chosen.